osquery/events/darwin/openbsm.cpp

```diff
@@ -29,7 +113,8 @@ Status OpenBSMEventPublisher::setUp() {
     LOG(WARNING) << "The auditpipe couldn't be opened.";
     return Status(1, "Could not open OpenBSM pipe");
   }
-  return Status(0);
+
+  return configureAuditPipe(audit_pipe_);
```

Comment:
Thanks for clarifying about the pipe configuration above.

I think we still want to apply a check for the configuration flag `audit_allow_config` to be consistent with Linux. Even though on macOS there are very few side-effects we should still be explicit when osquery is going to go 'beyond' the system configuration.

This means pulling

```
/// Control the audit subsystem by allowing subscriptions to apply rules.
FLAG(bool,
     audit_allow_config,
     false,
     "Allow the audit publisher to change auditing configuration");
```

Into `osquery/events/audit_flags.cpp` out from `osquery/events/linux/auditdnetlink.cpp` and then wrapping this line with a conditional check.